CHANGELOG.md

```diff
@@ -1,4 +1,32 @@
 ## 5.6.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_backup_dr_management_server` ([#6596](https://github.com/hashicorp/terraform-provider-google-beta/pull/6596))
+* **New Resource:** `google_compute_instance_settings` ([#6615](https://github.com/hashicorp/terraform-provider-google-beta/pull/6615))
+* **New Resource:** ``google_integration_connectors_connection`([#6612](https://github.com/hashicorp/terraform-provider-google-beta/pull/6612))
+
+IMPROVEMENTS:
+* assuredworkloads: added `enable_sovereign_controls`, `partner`, `partner_permissions`, `violation_notifications_enabled`, and several other output-only fields to `google_assured_workloads_workloads` ([#6597](https://github.com/hashicorp/terraform-provider-google-beta/pull/6597))
+* composer: added `storage_config` to `google_composer_environment` ([#6606](https://github.com/hashicorp/terraform-provider-google-beta/pull/6606))
+* container: added `fleet` field to `google_container_cluster` resource ([#6610](https://github.com/hashicorp/terraform-provider-google-beta/pull/6610))
+* container: marked `binary_authorization.evaluation_mode` field as computed and optional in `google_container_cluster` resource ([#6610](https://github.com/hashicorp/terraform-provider-google-beta/pull/6610))
```

Comment:
What does this mean for users?

Reply:
not much, going to remove. realistically it means that this can now support a computed value

Reply:
Needed due to fleet, but fleet is covered above. I don't think the impact is really necessary detail